### ISSUE TITLE
Usage of `--vpp-nvvfx-denoise` creates error in version 7.42

### ISSUE THREAD

**Issue by quamt** [rigaya/NVEnc#561]
Hello @rigaya 

When using the `--vpp-nvvfx-denoise` feature in version 7.42, an error message is displayed.
`nvvfx-denoise: Failed to allocate buffer for state: undeveloped feature..`

I reverted back to version 7.41, and the output appears as follows:
```
NVEncC (x64) 7.41 (r2681) by rigaya, Jan 22 2024 13:02:15 (VC 1929/Win)
OS Version     Windows 11 x64 (22631) [UTF-8]
CPU            12th Gen Intel Core i9-12900K [4.89GHz] (8P+8E,16C/24T)
GPU            #0: NVIDIA GeForce RTX 3080 (8704 cores, 1725 MHz)[PCIe4x16][551.61]
NVENC / CUDA   NVENC API 12.1, CUDA 12.4, schedule mode: auto
Input Buffers  CUDA, 46 frames
Input Info     avcuvid: H.264/AVC, 1920x1080, 24000/1001 fps
Trim           16000-20000 [offset: 0]
AVSync         vfr
Vpp Filters    cspconv(nv12 -> yv12(16bit))
               nvvfx-denoise: cspconv(yv12(16bit) -> bgr(fp32))
                              nvvfx-denoise: strength 0
                              cspconv(bgr(fp32) -> yv12(16bit))
               deband: mode 1, range 15, threY 15, threCb 15, threCr 15
                       ditherY 15, ditherC 15, blurFirst no, randEachFrame no
               cspconv(yv12(16bit) -> p010)
Output Info    H.265/HEVC main10 @ Level auto
               1920x1080p 1:1 23.976fps (24000/1001fps)
               avwriter: hevc, aac => matroska
Encoder Preset default
Rate Control   VBR
Multipass      2pass-full
Bitrate        0 kbps (Max: 24000 kbps)
Target Quality 25.00
Initial QP     I:20  P:23  B:25
QP range       I:0-63  P:0-63  B:0-63
QP Offset      cb:0  cr:0
VBV buf size   auto
Split Enc Mode auto
Lookahead      on, 32 frames, Adaptive I, B Insert
GOP length     240 frames
B frames       5 frames [ref mode: each]
Ref frames     8 frames, MultiRef L0:3 L1:3
AQ             off
CU max / min   auto / auto
Others         mv:Q-pel pic-struct buf-period
```

I tested both interlaced and progressive material.

**Comment by rainman74** [comment]
![image](https://github.com/rigaya/NVEnc/assets/16306963/095e510c-f58f-493a-af2a-2b19024fab67)

same here

**Comment by rigaya** [comment]
Thank you for your report, will be fixed in next release.

**Comment by quamt** [comment]
> Thank you for your report, will be fixed in next release.

Thank you.

**Comment by rigaya** [comment]
The problem should be fixed in [NVEnc 7.43](https://github.com/rigaya/NVEnc/releases/tag/7.43).

**Comment by quamt** [comment]
Thank, seem to be working as expected.
